src/federation/index.ts

```diff
@@ -109,6 +109,7 @@ if (getRedisUrl() == null) {
 export const federation = createFederation<void>({
   kv,
   queue,
+  allowPrivateAddress: process.env["ALLOW_PRIVATE_ADDRESS"] === "true"
```

Comment:
It's turned off to prevent [SSRF](https://owasp.org/www-community/attacks/Server_Side_Request_Forgery) attacks. Why do you need this option?

Reply:
> It's turned off to prevent [SSRF](https://owasp.org/www-community/attacks/Server_Side_Request_Forgery) attacks. Why do you need this option?

To build https://data.funfedi.dev/, I run all the Fediverse applications in a local docker network. It uses ips starting with 178.3.x.x by default, I think. These fall into the private addresses. So I need a way to disable this.

I'm not sure if I know how to check for an IP range here. Is it fine if I just add the variable to .env.sample?

Reply:
Okay, fair enough.  Please add it to the *.env.sample*.